bn-apps/memberships-management/membership-service/src/main/kotlin/net/corda/businessnetworks/membership/bno/GetMembershipsFlowResponder.kt

```diff
@@ -22,8 +22,6 @@ class GetMembershipListFlowResponder(flowSession : FlowSession) : BusinessNetwor
         //build memberships snapshot
         flowSession.receive<MembershipListRequest>()
         val databaseService = serviceHub.cordaService(DatabaseService::class.java)
-        val memberships = databaseService.getActiveMemberships()
-
-        flowSession.send(MembershipsListResponse(memberships))
+        flowSession.send(MembershipsListResponse(databaseService.getAllMemberships()))
```

Comment:
This should be more careful to actually only return the memberships where the node is a BNO. Otherwise it's going to break if the BNO itself is a member to another business network. Edge case, I know!

Reply:
That's a good catch. Added code to filter memberships where we are the BNO only before returning the snapshot